controllers/releaseplan/adapter_test.go

```diff
@@ -214,7 +214,8 @@ var _ = Describe("ReleasePlan adapter", Ordered, func() {
 			Spec: v1alpha1.ReleasePlanSpec{
 				Application:            application.Name,
 				Target:                 "default",
-				Pipeline:               parameterizedPipeline,
+				TenantPipeline:         parameterizedPipeline,
+				FinalPipeline:          parameterizedPipeline,
```

Comment:
The final pipeline is optional, I don't think you need this addition

Reply:
Removed